internal/configs/ingress.go

```diff
@@ -235,6 +235,7 @@ func createLocation(path string, upstream version1.Upstream, cfg *ConfigParams,
 		Upstream:             upstream,
 		ProxyConnectTimeout:  cfg.ProxyConnectTimeout,
 		ProxyReadTimeout:     cfg.ProxyReadTimeout,
+		ProxySendTimeout:     cfg.ProxySendTimeout,
```

Comment:
this is not required. see the comment in the `internal/configs/version1/config.go` file